.github/workflows/on-push-tag-build-without-cache.yml

```diff
@@ -0,0 +1,38 @@
+name: On Push Tag Build without Cache
+on:
+    push:
+        tags:
+            - "*"
+
+jobs:
+  docker:
+    name: Build Docker Tag without Cache
+    runs-on: ubuntu-20.04
+    environment: production
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Determine Meta
+        run: echo "REF=${GITHUB_REF##*/}" >> $GITHUB_ENV
+
+      - name: Dockerhub Login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Docker Build and Push (without cache)
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          build-args: |
+            GIT_REPO=https://github.com/${{ github.repository }}.git
+            GIT_REF=${{ env.REF }}
+          tags: |
+            ${{ secrets.DOCKER_REPOSITORY }}:${{ env.REF }}
+            ${{ secrets.DOCKER_REPOSITORY }}:latest
```

Comment:
Is there some particular reason to store the docker repository in a repo secret?